browser/ui/webui/brave_education/education_page_handler_unitttest.cc

```diff
@@ -0,0 +1,157 @@
+/* Copyright (c) 2024 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include <memory>
+#include <optional>
+#include <string>
+#include <vector>
+
+#include "base/strings/string_number_conversions.h"
+#include "base/test/bind.h"
+#include "base/test/test_future.h"
+#include "brave/browser/ui/webui/brave_education/education_page_handler.h"
+#include "brave/components/brave_vpn/common/buildflags/buildflags.h"
+#include "chrome/test/base/scoped_testing_local_state.h"
+#include "chrome/test/base/testing_browser_process.h"
+#include "chrome/test/base/testing_profile.h"
+#include "content/public/test/browser_task_environment.h"
+#include "mojo/public/cpp/bindings/remote.h"
+#include "testing/gtest/include/gtest/gtest.h"
+
+namespace brave_education {
+
+class TestDelegate : public EducationPageHandler::Delegate {
```

Comment:
Do we have a browser test for the actions separately, where we confirm that `OpenRewardsPanel`, etc are doing their actions... I assume we do, but just checking.

Reply:
Yes - in `education_page_browsertest`.